### ISSUE TITLE
Logout without any activity?

### ISSUE THREAD

**Issue by balochlens** [ohld/igbot#1030]
```
λ python like_hashtags.py streetphotography
2019-09-06 00:25:24,696 - INFO - Instabot Started
We need to create a text file 'secret.txt' where we will store your login and password from Instagram.
Don't worry. It will be stored locally.
Enter your login: xxxxxxxxx
Enter your password: (it will not be shown due to security reasons - just start typing and press Enter)
Password:
Do you want to add another account? (y/n) n
Which account do you want to use? (Type number)
1: xxxxxxxxx
0: add another account.
-1: delete all accounts.
1

```
It stuck here without any error.

**Comment by Oyetomi** [comment]
TroubleShoot :
1)make sure your script isn't scheduled 
2) CTRL + C (solved the issue)

**Comment by steffanjensen** [comment]
CTRL + C makes a false logout request to IG, so it's more safe way to stop the script, than ending a session. 

It could be this function has been taken out of this version of instabot or you should modify it yourself. 

Example to enable logout on CTRL + C:
```python 
bot = Bot(do_logout=True)
```
How I do it: https://github.com/instabotai/instabotai/blob/master/run.py#L22

**Comment by balochlens** [comment]
@virginsince1997  @reliefs 

I never used Scheduling.

I am using Windows 10, python 3.7

Well I did the followings:


Restarted my Laptop
Re-installed the bot.
run the command.....
but....
After doing all the exercises still on same place.

**Comment by balochlens** [comment]
Finally solved by uninstalling python 3.7 and installing python 3.6

**Comment by steffanjensen** [comment]
Problem should be solved and not closed with a "workaround".
